docs/api.md

```diff
@@ -19,6 +19,7 @@ follow a structure similar to superagent's request API itself.
 `request` object properties:
 
 - `url` *(String)*: the remote resource path. **required**
+- `prefix` *(String)*: prepends url with value, overrides driver option.
```

Comment:
Can I ask you to move these changes out of this file and into src/index.js? This file is auto-generated from that, so we never should directly modify this one.